ciw/simulation.py

```diff
@@ -86,6 +86,31 @@ def choose_tracker(self, tracker, deadlock_detector):
             return NaiveTracker(self)
         return StateTracker(self)
 
+    def detect_deadlock(self):
```

Comment:
@geraintpalmer this might have been added back in by me by mistake during the rebase back on to master (various merge conflicts etc). Let me know if this needs throwing out.